admin_scripts/ubuntu/change_computer_name.sh

```diff
@@ -11,11 +11,10 @@ fi
 
 NEW_NAME=$1
 PREFS_FILE=/etc/hosts
-STARTPAGE_PREFS_LINE="127.0.1.1 $NEW_NAME"
-OLD_LINE=$(grep 127.0.1.1 /etc/hosts)
-hostname $NEW_NAME
+OLD_NAME=$(hostname)
 
+hostname $NEW_NAME
 # Replace old line with new
-sed -i "s/$OLD_LINE/$STARTPAGE_PREFS_LINE/g" $PREFS_FILE
-
+sed -i "s/$OLD_NAME/$NEW_NAME/g" $PREFS_FILE
+echo $NEW_NAME > /etc/hostname
 
```

Comment:
As an observation, does this script also update `/etc/os2borgerpc/os2borgerpc.conf`? I don't see anything there.

(Perhaps the property in that file should become special data like `_os2borgerpc.client_version` so that it never gets out of sync?)

Reply:
I created a new ticket, #40783, to handle this.